components/ui/Input.tsx

```diff
@@ -0,0 +1,104 @@
+import {
+  StyleSheet,
+  TextInput,
+  TextInputProps,
+  TextStyle,
+  View,
+} from "react-native";
+import { InputVariant, SizeVariants } from "@/types";
+import { theme } from "@/theme";
+
+const inputSizeMap: Partial<Record<SizeVariants, TextStyle>> = {
+  lg: { fontSize: 20, lineHeight: 28 },
+  base: { fontSize: 16, lineHeight: 24 },
+  sm: { fontSize: 15, lineHeight: 28 },
+};
```

Comment:
If possible, could you compare these font sizes with those from the `customText` component and define them all in the theme file to ensure uniformity in the project?